config/boards/core3566.csc

```diff
@@ -0,0 +1,66 @@
+# Rockchip RK3566 quad core 4/8GB RAM SoC WIFI/BT eMMC USB2 USB3 NVMe PCIe GbE HDMI SPI
+BOARD_NAME="core3566"
+BOARDFAMILY="rk35xx"
+BOARD_MAINTAINER=""
+BOOTCONFIG="orangepi-3b-rk3566_defconfig"
+BOOT_SOC="rk3566"
+KERNEL_TARGET="vendor,edge"
```

Comment:
1.  I saw that you added support for linux-6.6 (current), but why isn't there any mention of support for `current` here?
2.  For `vendor` kernel support, you should submit the kernel device tree [here](https://github.com/armbian/linux-rockchip/tree/rk-6.1-rkr3/arch/arm64/boot/dts/rockchip).

Reply:
got it. will do.